utils/cautils/teams.go

```diff
@@ -23,16 +22,27 @@ type bootstrapAPIResponse struct {
 // BootstrapTeam does a request to api.smallstep.com to bootstrap the
 // configuration of the given team name.
 func BootstrapTeam(ctx *cli.Context, name string) error {
-	u, err := url.Parse(apiEndpoint)
-	if err != nil {
-		return errors.Wrapf(err, "error parsing %s", apiEndpoint)
+	apiEndpoint := ctx.String("endpoint")
+	if apiEndpoint == "" {
+		// Use the default endpoint..
+		u := url.URL{
+			Scheme: "https",
+			Host:   "api.smallstep.com",
+			Path:   "/v1/teams/" + name + "/cas/default",
+		}
```

Comment:
Anyone know off hand if URL has `Validate()` func?

Reply:
Not really, because it can build different kinds of URIs, you can do one after url.Parse()